libs/live-nft/src/api/simplehash.ts

```diff
@@ -42,24 +41,28 @@ type NftFetchOpts = {
    * spam filtering mode, defaults to a constant %
    */
   filters?: string;
+  /**
+   * spam filtering threshold, defaults to a constant %
+   */
+  threshold?: number;
 };
 const defaultOpts = {
   limit: PAGE_SIZE,
-  filters: `spam_score__lte%3D${SPAM_FILTER_THRESHOLD}`,
+  filters: `spam_score__lte%3D`,
```

Comment:
I think that we can use https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams wdyt?

Reply:
btw, that default seems invalid @mcayuelas-ledger ? it doesn't have a value, it's like `spam_score__lte=`